### PR TITLE
[Docs] Updating Getting started documentation from JIRA to Swift Issues

### PR DESCRIPTION
**Pull request by LucianoPAlmeida**
Just updating some links in the Getting Started documents.

### CODE REVIEW

File: docs/HowToGuides/FirstPullRequest.md [hunk 1]
```diff
@@ -159,10 +159,10 @@ to merge your changes. :tada:
 
 That's totally okay! There is no shame in that. You only have limited time and
 energy in a day. If you can, leave a comment on the bug report/pull request
-that you will not be able to continue and unassign yourself from the bug on
-JIRA. Don't worry about trying to explain _why_ you aren't able to contribute
-further. We understand. Unanticipated things come up all the time and you
-should do what _works for you_.
+that you will not be able to continue and unassign yourself from the issue on
+github. Don't worry about trying to explain _why_ you aren't
```
Comment (xedin) [inline docs/HowToGuides/FirstPullRequest.md:163]: Github have to be capitalized.

File: docs/HowToGuides/FirstPullRequest.md [hunk 2]
```diff
@@ -15,12 +15,12 @@ contribution process.
 
 In case you don't have something specific you'd like to work on, such as
 implementing something for a Swift Evolution pitch, you could start off by
-working on a bug labeled `StarterBug` on [Swift JIRA][StarterBug]. If the
-bug hasn't been assigned to someone, check the comments in case someone has
+working on a bug labeled `StarterBug` on [swift repository issues tab][StarterBug].
```
Comment (xedin) [inline docs/HowToGuides/FirstPullRequest.md:18]: Probably `Swift repository 'Issues' tab` is better?

**Comment by LucianoPAlmeida** [comment]
@swift-ci Please smoke test

**Review by xedin — APPROVED** [review]
LGTM, thank you!

**Pull request merged**